replicator-security/README.md

```diff
@@ -0,0 +1,128 @@
+# Replicator security configuration demos
+
+## Purpose 
+
+This repository provides docker-compose environments representing various Replicator security configurations. 
+
+## Non replicator components
+
+All environments contain:
+
+* A 3 node source cluster containing srcKafka1, srcKafka2 and srcKafka3
+* A 3 node destination cluster containing destKafka1, destKafka2, destKafka3
+* A single Connect worker that will run replicator
+* A source client container (srcKafkaClient) that creates a source topic and provides test data
+* A destination client container (destKafkaClient) that installs Replicator
+
+In all containers, minimal non-replicator configuration is supplied.
+
+## Test Data
+
+The data from testData/testData.txt is loaded into the source topic, this can be modified to suit testing requirements.
+
+## Creating and verifying the environments
+
+In all environments no actions are taken on the destination brokers. Because of this we can verify the environment by consuming from the test topic on the destination cluster.
+
+* Unsecured
+
+  This can be started with:
+  ```
+  docker-compose -f docker-compose_unsecure.yml up -d
+  ```
+  
+  And verified as follows:
+  ```
+  docker-compose -f docker-compose_unsecure.yml exec destKafka1 bash
+  kafka-console-consumer --bootstrap-server localhost:11091 --topic testTopic --from-beginning
+  ```
+
+* Source Cluster with SSL encryption
```

Comment:
This begs the question: source cluster has SSL, what does destination cluster have?  Perhaps for each demo, you identify both source & destination settings (maybe in a table?)

Reply:
I tried a table but it looked bad as there were a lot of gaps. Instead I called out the security under each demo header:

### Source Cluster with SASL Plain authentication

Source Cluster: SASL Authentication

Destination Cluster: Unsecure